client/blocks/reader-subscription-list-item/index.jsx

```diff
@@ -36,7 +36,6 @@ function ReaderSubscriptionListItem( {
 	const siteIcon = get( site, 'icon.img' );
 	const feedIcon = get( feed, 'image' );
 	const streamUrl = getStreamUrl( feedId, siteId );
-	const siteUrl = url || getSiteUrl( { feed, site } );
```

Comment:
I had wanted the component to ideally work when just given a feed or site.

Right now the only place taking advantage of that is devdocs I believe.